src/main/java/br/com/harbitech/school/filemanipulation/LoadDataSql.java

```diff
@@ -72,7 +72,7 @@ private void writeSubcategoriesInsert(List<SubCategory> subcategories, PrintStre
             SubCategoryStatus subcategoryStatus = sc.getStatus();
             String categoryCodeUrl = sc.getCategory().getCodeUrl();
 
-            String sqlSubcategory = "INSERT INTO Subcategory (name, code_url,order_visualization,description,status,category_code_url)"
+            String sqlSubcategory = "INSERT INTO Subcategory (name,code_url,order_visualization,description,status,code_url) SELECT code_url FROM Category"
```

Comment:
Acho que sobrou um `SELECT code_url FROM Category` aqui.

```suggestion
            String sqlSubcategory = "INSERT INTO Subcategory (name,code_url,order_visualization,description,status,code_url) "
```